stock_inventory_valuation_report/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     'name': 'Stock Inventory Valuation Report',
     'summary': 'Add report button on Inventory Valuation.',
-    'version': '12.0.1.1.1',
+    'version': '12.0.1.1.2',
```

Comment:
No need to change manually the version. The bot does it.

Reply:
Thanks for the review!
Can this manual change cause any issue? If not, I'd rather leave it

Reply:
It can conflict if other PR for the same module is done. Not usual, but a possibility.